video_xblock/static/css/handout.css

```diff
@@ -32,3 +32,7 @@
     background-color: #1aa1de;
     color: #fff;
 }
+
+.is-hidden {
```

Comment:
> it looks like a core style, not handout related.

We have only one css file included to student-view. Renamed to `student-view.css`
Core styles were moved to the top of file